src/adapter/src/catalog.rs

```diff
@@ -2743,14 +2698,31 @@ impl Default for DefaultPrivileges {
 impl DefaultPrivileges {
     /// Add a new default privilege into the set of all default privileges.
     fn grant(&mut self, object: DefaultPrivilegeObject, privilege: DefaultPrivilegeAclItem) {
+        if privilege.acl_mode.is_empty() {
```

Comment:
Why does this need to be here?

Reply:
It's just to be a bit defensive to avoid adding an empty privilege to our map. It should be unreachable since the planner/parser won't allow an empty grant set. I can remove it if you'd like.

Reply:
Nah keeping is fine. Was mostly curious.